### ISSUE TITLE
Missing accessibility add on hint - (Original story: Add accessibility hint to out-of-app web links)

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#25589]
## What happened?
Missing the accessibility add-on link for the following when the user's browsers opens outside of the app:

1. Active Claim detail > What should I do if I disagree... > Learn about your decision review options

## Specs:
- Device: Both Android and iOS

## Steps to Reproduce
Active screen reading software and navigate to the page / link.

## Desired behavior
Adds "This page will open in your device's browser" to accessibility link when the link opens up the users browser outside of the app.

## Acceptance Criteria
See above.

## Bug Severity
- 4 - Low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/24623

## Screen shot(s) and additional information
n/a

**Comment by digitalken** [comment]
6/25/2021 - Not hearing the off-site link announcement in iOS or Android.  Sending back to @keli13 for a look-see.

**Comment by digitalken** [comment]
7/8/2021 - Not in build as of this date.

**Comment by digitalken** [comment]
PASS 7/16/2021 - Sounds good on iOS and Android. Closing.
